buildSrc/subprojects/publishing/src/main/kotlin/gradlebuild/kotlin-dsl-plugin-bundle.gradle.kts

```diff
@@ -27,6 +27,10 @@ plugins {
 
 extensions.create<PluginPublishExtension>("pluginPublish", gradlePlugin, pluginBundle)
 
+tasks.withType<ValidatePlugins>() {
+    enableStricterValidation.set(true)
```

Comment:
IIUC, this is the only place where the `java-gradle-plugin` is applied? And then we don't need all the extra configuration from `task-properties-validation`?

Reply:
Yes. The setup done by the gradle-java-plugin should be sufficient. These are external plugins, which just happen to be part of the "core" build.